Library/Homebrew/formula.rb

```diff
@@ -1896,17 +1896,23 @@ def bottle_hash
     bottle_spec = stable.bottle_specification
     hash = {
       "rebuild"  => bottle_spec.rebuild,
-      "cellar"   => (cellar = bottle_spec.cellar).is_a?(Symbol) ? cellar.inspect : cellar,
-      "prefix"   => bottle_spec.prefix,
       "root_url" => bottle_spec.root_url,
       "files"    => {},
     }
     bottle_spec.collector.each_key do |os|
-      bottle_url = "#{bottle_spec.root_url}/#{Bottle::Filename.create(self, os, bottle_spec.rebuild).bintray}"
-      checksum = bottle_spec.collector[os][:checksum]
+      collector_os = bottle_spec.collector[os]
+      os_cellar = collector_os[:cellar]
+      os_cellar = os_cellar.is_a?(Symbol) ? os_cellar.inspect : os_cellar
+
+      checksum = collector_os[:checksum].hexdigest
+      filename = Bottle::Filename.create(self, os, bottle_spec.rebuild).bintray
+      path, = bottle_spec.path_resolved_basename(name, checksum, filename)
+      url = "#{bottle_spec.root_url}/#{path}"
```

Comment:
@MikeMcQuaid  I think this should be `path` without the comma, right? Now the bottle URL is missing the actual `path`